products/appengine/api.yaml

```diff
@@ -454,6 +454,11 @@ objects:
             required: true
             description: |
               The format should be a shell command that can be fed to bash -c.
+      - !ruby/object:Api::Type::Array
```

Comment:
I'd recommend making this a set instead (`is_set: true` in terraform.yaml) because the order these are specified in shouldn't matter.